go-client/client/example_table_ops_test.go

```diff
@@ -195,31 +195,33 @@ func doQueryOps() (string, error) {
 	// Table handles should be released when they are no longer needed.
 	defer baseTable.Release(ctx)
 
-	// Now, let's start building a query.
+	// Now, let's start building a query graph.
 	// Maybe I don't like companies whose names are too long or too short, so let's keep only the ones in the middle.
+	// Unlike with immediate operations, here midStocks is a QueryNode instead of an actual TableHandle.
+	// A QueryNode just holds a list of operations to be performed; it doesn't do anything until it's executed (see below).
```

Comment:
Great set of comments.  In the "immediate" section, do you want to note that the table is immediately returned?